### ISSUE TITLE
Add play.mvc.Http.Context.Implicit._ to default imports

### ISSUE THREAD

**Issue by Guidoch** [scala-ide/scala-ide-play2#257]
Hi,

the play! framework includes the following as default import for the html templates:
play.mvc.Http.Context.Implicit._

If someone agrees with me, I will add it to the default imports in the plugin and raise the pull request.

Thank you,
Guido

**Comment by sschaef** [comment]
What is the issue? Is this default import missing for the presentation compiler but not for the builder (i.e. one sees errors in the editor but the code compiles fine)?

**Comment by Guidoch** [comment]
Yes, the template editor doesn't find ctx(), session() or flash(), but when compiling and running there are no actual errors.

**Comment by sschaef** [comment]
If you want to fix this please go ahead. Contributions welcome!

**Comment by oxc** [comment]
I believe this was fixed in 4d425033b201a2a7c641f55f054da52941d8909f
